docs/content/concepts/assets/asset-checks/checking-for-data-freshness.mdx

```diff
@@ -356,6 +357,41 @@ height={1944}
 
 ---
 
+## Alert on overdue assets (Dagster+ only)
+
+In Dagster+, you can set up [alerts](/dagster-plus/managing-deployments/alerts) to notify you when an asset is overdue for an update. By default, a freshness check will fail with severity `WARN` if an asset is overdue for an update. Then, in order to alert on overdue assets, you'll want to create an alert policy that triggers on asset check failures with severity `WARN` (or `ERROR` if you've overridden the default severity), for your specific assets which you've defined freshness checks for. Asset alert policies can currently be targeted at specific asset keys, asset groups, or all assets.
+
+To become more familiar with the various ways alert policies can be set up, see the guide on [managing alert policies](/dagster-plus/managing-deployments/alerts/managing-alerts-in-ui).
+
+<TabGroup>
+<TabItem name="Option 1: Use the alert policy editor UI">
+
+1. Navigate to the **Deployments** in the top navigation
+2. Click the **Alerts** tab.
+3. Click **+Create alert policy**.
+4. Select the **Asset** alert policy type in the **Create alert policy** window.
+5. Select an asset (or set of assets) to target with the alert policy. You can target all
+
+assets, a specific asset key, or an asset group. 6. Check the box for **Failed (WARN)** or **Failed (ERROR)** severity, depending on the severity set for freshness check failure. 7. Select a [notification service](/dagster-plus/managing-deployments/alerts#supported-notification-services), provide the necessary information, and save the alert policy. </TabItem> <TabItem name="Option 2: from config">
```

Comment:
Some wonky formatting here - it should be something like this + running `make mdx-format` to get any nitpicky spacing:

```suggestion
5. Select an asset (or set of assets) to target with the alert policy. You can target all assets, a specific asset key, or an asset group.
6. Check the box for **Failed (WARN)** or **Failed (ERROR)** severity, depending on the severity set for freshness check failure.
7. Select a [notification service](/dagster-plus/managing-deployments/alerts#supported-notification-services), provide the necessary information, and save the alert policy.
</TabItem>
<TabItem name="Option 2: from config">
```